core/src/wallet/tezos/explorers/TezosLikeBlockchainExplorer.h

```diff
@@ -115,7 +115,7 @@ namespace ledger {
                                         const std::vector<std::string> &matchableKeys);
 
             virtual Future<std::shared_ptr<BigInt>>
-            getBalance(const std::vector<TezosLikeKeychain::Address> &addresses) = 0;
```

Comment:
Maybe we could keep `TezosLikeKeychain::Address` instead of `std::string` to keep some more type-safety?

Reply:
1- As you can see for all other functions in this class, address was always a string.
2- the old format was using a std::vector which must contais only one element (else an error)
So to keep the code simple and consistent, I changed the signature of this function